src/api/js-api.js

```diff
@@ -207,7 +207,13 @@ function init() {
               if (scriptEl.src) {
                 nextPromise = fetchExternalMap(scriptEl.src);
               } else {
-                nextPromise = Promise.resolve(JSON.parse(scriptEl.textContent));
+                let parsedContent;
+                try {
+                  parsedContent = JSON.parse(scriptEl.textContent);
+                } catch (e) {
+                  parsedContent = createEmptyImportMap();
+                }
+                nextPromise = Promise.resolve(parsedContent);
```

Comment:
Was this a bug in the original code?

Reply:
It was indeed:
https://github.com/single-spa/import-map-overrides/pull/87/files#r1264723158

Reply:
More of an unwanted behaviour.